core/src/main/java/hudson/model/MultiStageTimeSeries.java

```diff
@@ -101,9 +97,9 @@
     public MultiStageTimeSeries(Localizable title, Color color, float initialValue, float decay) {
         this.title = title;
         this.color = color;
-        this.sec10 = new TimeSeries(initialValue,decay,6*60);
-        this.min = new TimeSeries(initialValue,decay,60*24);
-        this.hour = new TimeSeries(initialValue,decay,28*24);
+        this.sec10 = new TimeSeries(initialValue, decay, 6 * (int) TimeUnit.HOURS.toMinutes(6));
```

Comment:
Looks to be a typo here

Reply:
What's the typo? This is a "10 second" interval time series, so we need 6x the data points of whatever time span we want to record in minutes.

Reply:
Ahh I was mistaken then, the `6 *` wasn't immediately obvious to me